buildSrc/src/main/java/com/uber/okbuck/core/task/OkBuckTask.java

```diff
@@ -46,7 +45,7 @@ void okbuck() {
     }
 
     // Fetch Kotlin support deps if needed
-    boolean hasKotlinLib = okBuckExtension.buckProjects.parallelStream().anyMatch(project -> ProjectUtil.getType(project) == ProjectType.KOTLIN_LIB);
+    boolean hasKotlinLib = ProjectUtil.hasArtifactInClasspath(getProject(), KotlinUtil.KOTLIN_GROUP, KotlinUtil.KOTLIN_GRADLE_MODULE);
```

Comment:
Can we move this utility method into KotlinUtil instead? Its weird to put this on project util if its not reused anywhere else